### ISSUE TITLE
Incorrect version output in release 0.4.11b7

### ISSUE THREAD

**Issue by amanagr** [semgrep/semgrep#536]
After extracting files via:
```
version=0.4.11b7
tarball=semgrep-v$version-ubuntu-16.04.tgz
tarball_url=https://github.com/returntocorp/sgrep/releases/download/v$version/$tarball
wget -nv "$tarball_url"
tar -xzf "$tarball"
```
and running:
```
./sgrep-lint-files/sgrep-lint --version
```
outputs: `0.4.9`

**Expected behavior**
version should be `0.4.11b7`

**Comment by dlukeomalley** [comment]
@amanagr We'll cut a release shortly to address this, thank you for the ticket.

**Comment by underyx** [comment]
Hey @amanagr, we've just released `0.4.11b8` with the correct `--version` output. You can get it with these install script parameters:

```bash
version=0.4.11b8
tarball=semgrep-v$version-ubuntu-16.04.tgz
sha256=19577b17855da89d300e92633f630bd6f1cb855071408a1611fe301bddffe959
tarball_url=https://github.com/returntocorp/semgrep/releases/download/v$version/$tarball
```

Sorry about the inconvenience! You might've seen above, we plan to add a safeguard to not be able to make this mistake again: https://github.com/returntocorp/semgrep/issues/538

**Comment by amanagr** [comment]
Thanks for the quick fix.
